apps/src/javalab/JavalabView.jsx

```diff
@@ -126,6 +215,10 @@ class JavalabView extends React.Component {
       document.body.style.backgroundColor = color.background_gray;
     }
 
+    // It's possible that a console level without instructions won't have
+    // anything to show on the left side.
+    const leftSideVisible = appType !== 'console' || !!longInstructions;
```

Comment:
Trying to remember... is there also a field for `shortInstructions`?

Reply:
I'm not too familiar with all our instructional variants, but from looking at `TopInstructions.jsx`, I only saw `longInstructions` being passed into `Instructions`.

Reply:
`shortInstructions` is only used for CSF courses and I guess only passed into `CSFInstructions` and not `TopInstructions`.